src/openrct2/world/MoneyEffect.cpp

```diff
@@ -47,16 +47,16 @@ void MoneyEffect::CreateAt(money32 value, int32_t x, int32_t y, int32_t z, bool
     if (moneyEffect == nullptr)
         return;
 
-    moneyEffect->value = value;
-    moneyEffect->vertical = (vertical ? 1 : 0);
+    moneyEffect->Value = value;
+    moneyEffect->Vertical = (vertical ? 1 : 0);
```

Comment:
I believe that vertical is actually whether it is a credit or debit. We should make this an enum class and define the two states.

Reply:
I think it has two meanings, I did convert it to an enum, but the `GetStringId` makes things look a bit weird. I've added the commit for you guys to see